include/glow/Base/Tensor.h

```diff
@@ -187,6 +191,13 @@ class Tensor final {
   /// \returns the number of elements in the tensor.
   size_t size() const { return type_.size(); }
 
+  /// \returns the actual number of elements in the tensor taking striding into
+  /// account. Since size() does not take striding into account, size() is
+  /// always <= actualSize().
+  size_t actualSize() const {
```

Comment:
This is pretty confusing unless you go look at what these do. If I used `size()` not knowing about `actualSize()` I would be confused, I'm not sure what a better naming scheme would be though. Is there anything similar we could look to for names>?

Reply:
The proposal is to have intuitive names for cases, when we don't want to worry about garbage data: `size()`, `at()`, iteration.
If user knows what they're doing, they can use `raw()`, `actualSize()`.